utils/mock-data-loader/gef/mandatoryCriteriaVersioned.js

```diff
@@ -1,9 +1,9 @@
 const MANDATORY_CRITERIA_VERSIONED = [
   {
-    version: 33,
-    createdAt: new Date('2021-01-03T00:00'),
+    version: 3,
```

Comment:
Shouldn't this be version 34?

Reply:
Not for `GEF` but yes `34` for `BSS/EWCS`.

Reply:
Got it, thanks